tools/test.py

```diff
@@ -697,7 +697,7 @@ def Execute(args, context, timeout=None, env={}, faketty=False):
 
 
 def ExecuteNoCapture(args, context, timeout=None):
-  (process, exit_code, timed_out) = RunProcess(
+  (process, exit_code, timed_out, _) = RunProcess(
```

Comment:
This function is basically dead code.  I'd be alright with removing it and the code path leading up to it (the `if not options.no_build:` statement in Main().)